Fixed:Implementation of Qr code Dynamically

GOKULGSECE opened the pull request:
Fixed: #138
I have implemented a QR code generator dynamically based upon the user's input hereby attaching the screenshot /videoof the same

https://github.com/oxiton-foundation/click-metrics/assets/125350820/637789a4-8357-4c5d-a7de-449de880a5bd

Thanks for assigning the issue to me

Swarnendu0123 commented on client/src/components/pages/App.css:
```diff
@@ -0,0 +1,99 @@
+/* App.css */
```
Don't use inline css. Use tailwind.

GOKULGSECE replied:
Ok I will implement tailwind In this project

Swarnendu0123 commented:
There arenfew suggesions

- By default we can render the qr of click matrics.
- Use buttons in black.
- Decrease the width of the input.
- Buttons can be on the same horizontal level.

Swarnendu0123 commented on client/package.json:
```diff
@@ -16,6 +16,7 @@
     "@radix-ui/react-label": "^2.0.2",
     "@radix-ui/react-slot": "^1.0.2",
     "antd": "^5.18.0",
+    "antf": "^1.0.0",
```
What is the function of this new library?

GOKULGSECE replied:
As we can get a message stating that the QR haa been generated as a popup

Swarnendu0123 commented on .hintrc:
```diff
@@ -0,0 +1,13 @@
+{
```
Why did you add this file? Is there a specific purpose of this file?

GOKULGSECE replied:
Actually while committing the changes via a message this file is getting edited

Swarnendu0123 commented:
There arenfew suggesions

- By default we can render the qr of click matrics.
- Use buttons in black.
- Decrease the width of the input.
- Buttons can be on the same horizontal level.
- You should write Fixes #issue_number because if your PR gets merged the issue will be closed, otherwise we have to manually close it.

GOKULGSECE commented:
Oke I will resolve the reviews and raise a PR accordingly

GOKULGSECE commented:
i have re-checked and requested for the PR as i have implemented Tailwind CSS for the following and i have changed as per your request Thank you for Assigning me this issue

Swarnendu0123 commented on client/src/components/pages/qr-codes.tsx:
```diff
@@ -1,31 +1,96 @@
+        message.error("Input cannot be empty");
+        return;
+      }
+      const url = `https://api.qrserver.com/v1/create-qr-code/?size=150x150&data=${encodeURIComponent(
```
What is the purpose of this url?

GOKULGSECE replied:
It helps to generate the QRcode dynamically ,using this url it can be able to generate QR code with respect to the input given by the user. Basically it is like a API for generating QR code generation

Swarnendu0123 commented:
Can you please attach one video after the final changes you have done?

GOKULGSECE commented:
HEreby i am attaching the video for my implementation

https://github.com/oxiton-foundation/click-metrics/assets/125350820/8aa7cd11-a362-4cb0-ad22-0bfe0340451e

Swarnendu0123 commented:
Thabks:) Great Work!
